sdk/Lusid.Sdk.Tests/tutorials/Instruments/DemoEquityOption.cs

```diff
@@ -0,0 +1,181 @@
+            Assert.That(roundTripEquityOption.DomCcy, Is.EqualTo(equityOption.DomCcy));
+            Assert.That(roundTripEquityOption.OptionType, Is.EqualTo(equityOption.OptionType));
+            Assert.That(roundTripEquityOption.StartDate, Is.EqualTo(equityOption.StartDate));
+            Assert.That(roundTripEquityOption.OptionMaturityDate, Is.EqualTo(equityOption.OptionMaturityDate));
+            Assert.That(roundTripEquityOption.OptionSettlementDate, Is.EqualTo(equityOption.OptionSettlementDate));
+            Assert.That(roundTripEquityOption.UnderlyingIdentifier, Is.EqualTo(equityOption.UnderlyingIdentifier));
+        }
+
+        [Test]
+        public void DemoEquityOptionValuation()
+        {
+            var scope = "DemoEquityOptionValuation";
+
+            // upsert option
+            var option = InstrumentExamples.CreateExampleEquityOption();
+            string uniqueId = "id-equityOption-1";
+            _testDataUtilities.UpsertOtcToLusid(option, "some-name-for-this-equityOption", uniqueId);
+
+            // for Black-Scholes pricing, we need the following market data
+            _testDataUtilities.CreateAndUpsertSimpleQuote(scope, "ACME", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 110m, "USD", _demoEffectiveAt);
+            _testDataUtilities.CreateAndUpsertOisCurve(scope, _demoEffectiveAt, "USD");
+            _testDataUtilities.CreateAndUpsertVolSurface(scope, _demoEffectiveAt, option, 0.2m);
+
+            // create Black-Scholes recipe specifying where to look for market data and which metrics to return
+            // if in a larger portfolio, we would make a specific VendorModelRule specifying that equity options are to be valued using Black-Scholes
+            var recipeCode = "EquityOption_ValuationRecipe";
+            var pricingOptions = new PricingOptions(new ModelSelection(ModelSelection.LibraryEnum.Lusid, ModelSelection.ModelEnum.BlackScholes));
+            var recipe = new ConfigurationRecipe
+            (
+                scope: scope,
+                code: recipeCode,
+                market: new MarketContext(
+                    new List<MarketDataKeyRule> {},
+                    options: new MarketOptions(defaultSupplier: "Lusid", defaultScope: scope, defaultInstrumentCodeType: "RIC")
+                    ),
+                pricing: new PricingContext(options: pricingOptions)
+            );
+            _testDataUtilities.UpsertRecipe(recipe);
+
+            // define the metrics that we wish to return
+            string ValuationDateKey = "Analytic/default/ValuationDate";
+            string InstrumentTag = "Analytic/default/InstrumentTag";
+            string HoldingPvKey = "Holding/default/PV";
+            var valuationSpec = new List<AggregateSpec>
+            {
+                new AggregateSpec(ValuationDateKey, AggregateSpec.OpEnum.Value),
+                new AggregateSpec(InstrumentTag, AggregateSpec.OpEnum.Value),
```

Comment:
you don't use the InstrumentTag nor Date? Omit to focus on the core aspect of merge?

Reply:
I consider it important metadata. Date, instrument name, pv. Would agree that if there were anything else that it should be removed for simplicity.